pkg/virt-config/feature-gates.go

```diff
@@ -172,3 +163,50 @@ func (config *ClusterConfig) AlignCPUsEnabled() bool {
 func (config *ClusterConfig) NodeRestrictionEnabled() bool {
 	return config.isFeatureGateEnabled(featuregate.NodeRestrictionGate)
 }
+
+func GetEnabledFeatureGates(kubevirtDevConfig *v1.DeveloperConfiguration) []string {
+	if kubevirtDevConfig == nil || (kubevirtDevConfig.FeatureGates == nil && kubevirtDevConfig.FeatureGatesMap == nil) {
+		return nil
+	}
+
+	featureGateMap := maps.Clone(kubevirtDevConfig.FeatureGatesMap)
+	if featureGateMap == nil {
+		return slices.Clone(kubevirtDevConfig.FeatureGates)
+	}
+
+	for _, fgFromList := range kubevirtDevConfig.FeatureGates {
+		if _, existsInMap := featureGateMap[fgFromList]; !existsInMap {
+			featureGateMap[fgFromList] = true
+		}
+	}
+
+	enabledFeatureGates := make([]string, 0, len(featureGateMap))
+	for fg, enabled := range featureGateMap {
+		if enabled {
+			enabledFeatureGates = append(enabledFeatureGates, fg)
+		}
+	}
+
+	return enabledFeatureGates
+}
+
+func IsFeatureGateEnabled(featureGate string, kubevirtDevConfig *v1.DeveloperConfiguration) bool {
+	if kubevirtDevConfig == nil {
+		return false
+	}
+
+	if fg := featuregate.FeatureGateInfo(featureGate); fg != nil && fg.State == featuregate.GA {
+		return true
+	}
+
+	if isEnabled, exists := kubevirtDevConfig.FeatureGatesMap[featureGate]; exists {
+		return isEnabled
+	}
+
+	for _, fg := range kubevirtDevConfig.FeatureGates {
+		if fg == featureGate {
+			return true
+		}
+	}
```

Comment:
Consider using the standard library for this kind of loops:
```suggestion
	return slices.Contains(config.GetConfig().DeveloperConfiguration.FeatureGates, featureGate)
```